samples/net/sockets/echo_server/src/echo-server.c

```diff
@@ -44,6 +46,24 @@ static void init_app(void)
 	k_sem_init(&quit_lock, 0, UINT_MAX);
 
 	NET_INFO(APP_BANNER);
+
+#if defined(CONFIG_NET_SOCKETS_SOCKOPT_TLS)
+	int err = tls_credential_add(PUBLIC_CERTIFICATE_TAG,
+				     TLS_CREDENTIAL_PUBLIC_CERTIFICATE,
```

Comment:
Maybe TLS_CREDENTIAL_SERVER_CERTIFICATE would be a better name for this after all?